README-docker-compose.md

```diff
@@ -0,0 +1,118 @@
+## Adding new extensions
+When adding a new extension you will need to add it properly in `setup.py` and then run a build.
+
+1. In `setup.py` add your new extension in the form of:
+```python
+'.<extension-name> = mfr.extensions.<extension-name>:<renderer-or-exporter-name>',
+
+```
+Make sure you place it under the correct `mfr.exporters` or `mfr.renderers` section.
+
+2. Make sure your docker-sync is running.
+
+3. Run
```

Comment:
Let's mention both `invoke install` and `python setup.py build`. Please add a link to how to setup `virtualenv` since we want OSF, WB, and MFR has each own isolated environment.